CMakeLists.txt

```diff
@@ -56,7 +56,8 @@ set(CMAKE_AUTOMOC ON)
 find_package(Qt5Core 5.0 REQUIRED)
 find_package(Qt5Widgets 5.0 REQUIRED)
 find_package(Qt5Multimedia 5.0 REQUIRED)
-#find_package(QT5OpenGL 5.0 REQUIRED)
+find_package(Qt5MultimediaWidgets 5.0 REQUIRED)
```

Comment:
If this is only needed by demodatv I would rather move it there. However this is not part of the target link libraries today is this new to Qt 5.11 ?

Reply:
Also sounds good. I'll move it out.